1.20/README.md

```diff
@@ -1,9 +1,8 @@
 Nginx 1.20 server and a reverse proxy server container image
 ============================================================
 This container image includes Nginx 1.20 server and a reverse server for OpenShift and general usage.
-Users can choose between RHEL, CentOS, CentOS Stream and Fedora based images.
+Users can choose between RHEL, CentOS Stream and Fedora based images.
 The RHEL images are available in the [Red Hat Container Catalog](https://access.redhat.com/containers/),
-the CentOS images are available in the [Quay.io](https://quay.io/organization/centos7),
```

Comment:
I would keep these two lines - the images are still available to download and we do expect them to be used until the EOL of centos7.